platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygComposerView/WysiwygComposerViewModel.swift

```diff
@@ -451,9 +451,16 @@ private extension WysiwygComposerViewModel {
             let attributed = try HTMLParser.parse(html: html,
                                                   style: parserStyle,
                                                   mentionReplacer: mentionReplacer)
-            // FIXME: handle error for out of bounds index
             let htmlSelection = NSRange(location: Int(start), length: Int(end - start))
```

Comment:
Should we check that start <= end?

Reply:
Not sure this will ever happen. 
But if it does I'm happy to go with the "old branch" which in the end should throw an error. 👍🏻